### PR TITLE
fix(artifacts test): make docker job run on master branch

### PR DESCRIPTION
**Pull request by enaydanov**
Trello: https://trello.com/c/UX7nHf02/1736-artifacts-docker-doesnt-work-in-master-branch-1908

Fixes: #1908 

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [x] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: docs/artifacts_test.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ jenkins-pipelines
 You should use different parameters for .rpm/.deb tests and for AMI test:
 - The only required parameter for .rpm/.deb jobs is `scylla_repo`, a path to a ScyllaDB repo (e.g., `https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/master/latest/scylla.repo`)
 - For AMI job you need two parameters: `scylla_ami_id` and `region_name`
-- For Docker job you need two parameters: `scylla_docker_image` and `scylla_version`
+- For Docker job you need three parameters: `scylla_docker_image` (e.g., `scylladb/scylla` or `scylladb/scylla-nightly`), `scylla_version` (it'll be used as Docker image tag) and `scylla_repo`
```
Comment (fruch) [inline docs/artifacts_test.md]: Why do we need `scylla_repo` ? for downloading debug symbols ?, I would argue we don't need it in artifact test.

And if we really need it, and it not configured in advance in the docker itself then we should look it up.

File: test-cases/artifacts/docker.yaml [hunk 2]
```diff
@@ -6,7 +6,8 @@ n_loaders: 0
 n_monitor_nodes: 0
 nemesis_class_name: 'NoOpMonkey'
 scylla_linux_distro: 'centos'
-scylla_version: '3.3.rc1'
+scylla_version: '3.3.rc2'
```
Comment (fruch) [inline test-cases/artifacts/docker.yaml]: Can we not hardcode those values ? It can lead to mistakes. 
Like pulling the wrong debug symbols.

Comment (enaydanov) [reply test-cases/artifacts/docker.yaml]: It's more like example. This parameter will be overridden by Jenkins job.

File: docs/artifacts_test.md [hunk 3]
```diff
@@ -118,7 +118,7 @@ SCT_AMI_ID_DB_SCYLLA="<ami you've found>" SCT_REGION_NAME=us-east-1 hydra run-te
 If you want to run against some official ScyllaDB Docker image, you should go to [tag list](https://hub.docker.com/r/scylladb/scylla/tags) and choose some tag name (i.e., `3.3.rc1`)
 
 ```sh
-SCT_SCYLLA_VERSION=<tag you've chose> hydra run-test artifacts_test --backend docker --config test-cases/artifacts/docker.yaml
+SCT_SCYLLA_VERSION="<tag you've chose>" SCT_SCYLLA_REPO="<URL to repo which used to build Docker image>" hydra run-test artifacts_test --backend docker --config test-cases/artifacts/docker.yaml
```
Comment (amoskong) [inline docs/artifacts_test.md]: The repo file exists in the docker file system `/etc/yum.repos.d/scylla.repo`.
We don't need to assign it by the SCT parameter.

Sometimes, we only know the docker tag, but we don't know the repo url which is used to build the Docker image.

Comment (enaydanov) [reply docs/artifacts_test.md]: This is great. It'll simplify things so much.

**Comment by enaydanov** [comment]
@fruch and @amoskong, thank you guys for making it much cleaner change.

File: docs/artifacts_test.md (hunk 1 above)
Comment (enaydanov) [reply docs/artifacts_test.md]: Removed this requirement. Not sure if we need to run SCT against bare OS as a Docker container, but for now it's not supported anyway.

**Pull request merged**